docker-compose-ubuntu.yaml

```diff
@@ -0,0 +1,24 @@
+version: '3.8'
+services:
+  graphdb:
+    container_name: graphdb
+    image: khaller/graphdb-free:latest
+    network_mode: host
+
+  ot-node:
+    image: origintrail/ot-node:v6_release_testnet_ubuntu
```

Comment:
Let's use proper name for image like we discussed on the thread

Reply:
now there is a conflict here in the naming format. According to @djordjekovac he recommends we follow origintrail/ot-node:v6_release_testnet_ubuntu as opposed to ot-node:v6.0.0-beta.1-debian. Because during image build it will always build with the latest so no need to specify tags as beta.